server/IdentityService/Utils/PasswordUtil.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace IdentityService.Utils
+{
+    public static class PasswordUtil
+    {
+        public static string CreatePasswordHash(string password)
+        {
+            var sha256 = SHA256.Create();
+            return HashToString(sha256.ComputeHash(Encoding.UTF8.GetBytes(password)));
+        }
+
+        private static string HashToString(IEnumerable<byte> hash)
```

Comment:
I find 
``` 
private static string HashToString(IEnumerable<byte> hash) =>
            hash
                .Aggregate(new StringBuilder(), (sb, b) => sb.Append(b.ToString("x2")))
                .ToString();
```
neat :D

Reply:
Correct me if I do not understand it correctly: in Aggregate first parameter is base value, second is function, which is executed for every element of enumarable?

Reply:
Correct :)